Code/GraphMol/RWMol.h

```diff
@@ -46,6 +46,7 @@ class RDKIT_GRAPHMOL_EXPORT RWMol : public ROMol {
       : ROMol(other, quickCopy, confId) {
     d_partialBonds.clear();
   };
+  RWMol(const RWMol &other) : ROMol(other){};
```

Comment:
Not that I understand it, but all the other ctors have d_partialBonds.clear().

Reply:
I'm assuming that this (d_partialBonds.clear()) isn't necessary in the "default" constructor.

Reply:
I believe that you're guaranteed that the unitialized container is empty.